libraries/AP_CANManager/AP_SLCANIface.h

```diff
@@ -50,8 +50,11 @@ class CANIface: public AP_HAL::CANIface
     bool handle_FrameRTRStd(const char* cmd);
     bool handle_FrameRTRExt(const char* cmd);
     bool handle_FrameDataStd(const char* cmd);
-    bool handle_FrameDataExt(const char* cmd);
+    bool handle_FrameDataExt(const char* cmd, bool canfd);
 
+#if HAL_CANFD_SUPPORTED
```

Comment:
need to make header same for with/without HAL_CANFD

Reply:
done